pkgs/by-name/pr/pretix/package.nix

```diff
@@ -260,6 +260,11 @@ python.pkgs.buildPythonApplication rec {
     "test_same_day_spanish"
     "test_same_month_spanish"
     "test_same_year_spanish"
+
+    # AssertionError
```

Comment:
I am bisecting these since 72 hours ago and putting `AssertionError` here as the cause and not knowing if they break anything in production is insufficient for me.